### PR TITLE
Implementation instructions

### PR DESCRIPTION
**Pull request by stellakunzang**
**Fixes issue #:** 

- [X] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

**Description:**
Fix seed file 
Begin writing README
Replace test objects with factories

**What does this solve?:**
DB seed will now work with no errors 
Instructions for FE for getting app working locally 
Test looks tidier

**Notes:**

- [X] All tests are passing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
 
-## Implementation Instructions 
-(Tailor to someone using rails for the first time) 
+## Introduction
+
+This application is a Ruby on Rails API service, utilizing GraphQL to create user-friendly endpoints. Our database is built using Postgres. You can find our schema in the section below.
+
+## Schema
+
+## Endpoints
```
Comment (stellakunzang) [inline README.md:9]: These are placeholders that will get filled in later once everything is complete.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+
+*If there is a problem with the database and you need to scrap and start over:
+
+- `$ rake db:drop`
+
+Then run the three `db` commands above.
+
+### Troubleshooting
+
+If the `db:seed` command works, there is no return value. Pay attention to what comes back because you may not realize that your seeding has failed. You can track down errors in the `db/seeds` file. The seeds are generated using FactoryBot, and the specifications are found in `spec/factories`. You can find our more about Faker [here](https://github.com/faker-ruby/faker) and FactoryBot [here](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md).
```
Comment (stellakunzang) [inline README.md:49]: These notes will hopefully be helpful to front end

File: config/initializers/cors.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
+    # in production change origins to heroku site
```
Comment (stellakunzang) [inline config/initializers/cors.rb:3]: Saw this online while researching CORS...for development `*` is fine but for production we will want to update this. I think the heroku site we will list here is front end but not 100% sure.

File: README.md (hunk 2 above)
Comment (elguapogordo) [reply README.md:49]: they're also good FYI info to anyone else viewing the repo. i did something similar on one of my last projects, although not to this extent

File: db/seeds.rb [hunk 4]
```diff
@@ -22,9 +22,8 @@
...
-StreetArt.create!({})
+# user = User.create(username: "Hank Hill", email: "ProPAIN@aol.com")
+#
+# user.street_arts.create(latitude: "39.7666", longitude: "104.9837", address: "RiNo", city: "Denver", state: "CO", zipcode: "80216", image_urls: ["https://cdn.5280.com/2017/04/RiNo-neighborhood-street-art.jpg"], description: "neat-o!", artist_name: "Larry", art_name: "cool-art-1", favorite: true, visited: true)
+# user.street_arts.create(latitude: "39.7527", longitude: "105.0017", address: "LoDo", city: "Denver", state: "CO", zipcode: "80202", image_urls: ["https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTW8zd6DEZ5UHfY8r8NJB-jEcpGAlbHlwg6Cg&usqp=CAU"], description: "neat-o!", artist_name: "Larry",  art_name: "cool-art-2", instagram_handle: "@insta", favorite: false, visited: true)
+# user.street_arts.create(latitude: "39.7581", longitude: "104.9874", address: "Five Points", city: "Denver", state: "CO", zipcode: "80216", image_urls: ["https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRe-qGNr3_dvdgNqE6g81qa4OTlSvmfbiGCRw&usqp=CAU"], description: "woww!", artist_name: "Sammi", instagram_handle: "@insta", favorite: true, visited: true)
```
Comment (Gallup93) [inline db/seeds.rb:29]: Maybe we should wrap the image array for these seeds into a string?

File: spec/requests/street_art_request_spec.rb [hunk 5]
```diff
@@ -4,10 +4,11 @@
...
-    @user.street_arts.create(latitude: "39.7527", longitude: "105.0017", address: "LoDo", city: "Denver", state: "CO", zipcode: "80202", image_urls: ["https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTW8zd6DEZ5UHfY8r8NJB-jEcpGAlbHlwg6Cg&usqp=CAU"], description: "neat-o!", artist_name: "Larry",  art_name: "cool-art-2", instagram_handle: "@insta", favorite: false, visited: true)
-    @user.street_arts.create(latitude: "39.7581", longitude: "104.9874", address: "Five Points", city: "Denver", state: "CO", zipcode: "80216", image_urls: ["https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRe-qGNr3_dvdgNqE6g81qa4OTlSvmfbiGCRw&usqp=CAU"], description: "neat-o!", artist_name: "Larry", instagram_handle: "@insta", favorite: true, visited: true)
+    @user = FactoryBot.create(:user)
+    @art_post = FactoryBot.create(:street_art, user: @user)
+    2.times do
+      FactoryBot.create(:street_art, user: @user)
+    end
```
Comment (elguapogordo) [inline spec/requests/street_art_request_spec.rb:11]: clean

**Review by elguapogordo — APPROVED** [review]

**Review by Gallup93 — APPROVED** [review]
Thank you Stella!

**Pull request merged**